creme/creme_core/tests/gui/test_menu.py

```diff
@@ -274,6 +280,269 @@ class TestEntry02(TestEntry01):
 
         self.assertHTMLEqual(expected, TestEntry02().render(ctxt))
 
+    def test_template_entry(self):
+        user = self.login_as_standard(admin_4_apps=['creme_config'])
```

Comment:
I don't think it's useful to login (user is injected explicitly in the context).
I propose:
`user = self.create_user(role=self.create_role(admin_4_apps=['creme_config']))`